src/pages/ChooseClass/ChooseClass.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import './chooseclass.scss'
+import dummyData from './dummyData.json'
+import { useHistory } from "react-router-dom";
+//import classes from '*.module.css';
+
+const ChooseClass = () => {
+ 
+  let history = useHistory();
+  function handleClick() {
+    history.push("/students-tracker");
+  }
+  return(
+    <div>
+      <div>
+        <h3>Welcome back <span>{dummyData[0].userName} </span>!</h3>
+        <p>Please choose which class would you like to access:</p>
+      </div>
+      <div>
+        <button type="button" onClick={handleClick}>
```

Comment:
You can map the dummyData array and write less code

Reply:
Mapping done